### DISCUSSION TITLE
lefthook install fails when no git is available

### DISCUSSION THREAD

**Discussion by JuliusHenke** [evilmartians/lefthook#348] [Q&A] [answered]
I currently need to install a project in a `node:16-alpine` CI environment and the `npm install` fails with:
```bash
> ... postinstall
> lefthook install -f
Error: exec: "git": executable file not found in $PATH
npm ERR! code 1
```

Could the lefthook install rather print a warning than fail with an error if no `git` executable was found?
An easy option to disable the lefthook install in CI could also be provided via the environment variable `LEFTHOOK=0`.

**Comment by mrexox** [comment, marked as answer]
You can also set the `CI` variable in the container: [see docs for disabling lefthook install in CI](https://github.com/evilmartians/lefthook/blob/master/docs/usage.md#disable-lefthook-in-ci)
